### PR TITLE
Branch-UI

### PR DESCRIPTION
**Pull request by rachelljt**

### CODE REVIEW

File: src/main/resources/view/TaskListCard.fxml [hunk 1]
```diff
@@ -6,27 +6,31 @@
...
+            <HBox alignment="CENTER_LEFT" spacing="5">
                 <Label fx:id="id" styleClass="cell_big_label">
                     <minWidth>
                         <!-- Ensures that the label text is never truncated -->
                         <Region fx:constant="USE_PREF_SIZE" />
                     </minWidth>
                 </Label>
-                <Label fx:id="title" text="\$first" styleClass="cell_big_label" />
+                <Label fx:id="title" styleClass="cell_big_label" text="\$first" />
             </HBox>
```
Comment (zatkiller) [inline src/main/resources/view/TaskListCard.fxml:29]: I think you might be missing the task status for the Card

**Review by zatkiller — CHANGES_REQUESTED** [review]
Just a small issue, but overall the UI looks good :D

File: src/main/resources/view/TaskListCard.fxml (hunk 1 above)
Comment (rachelljt) [reply src/main/resources/view/TaskListCard.fxml:29]: ohh i think i pulled this before u merged ur pr... so i dont have the task status class right now.. what shld i do?

Comment (rachelljt) [reply src/main/resources/view/TaskListCard.fxml:29]: oh its okay i got it!

**Review by zatkiller — APPROVED** [review]
LGTM. Next time click the rerequest review so that i get a notification and i can come to approve

**Pull request merged**